server/lib/pbench/indexer.py

```diff
@@ -3317,8 +3681,99 @@ def hostnames_if_ip_from_sosreport(sos_file_name):
     return (0, d)
 
 
+class BadIterationName(Exception):
```

Comment:
Can this be grouped with the other classes that raise an exception, it starts around line 93.

Reply:
Sure.